main/bind/APKBUILD

```diff
@@ -101,7 +107,6 @@ build() {
 		--with-libxml2 \
 		--with-openssl=/usr \
 		--with-randomdev=/dev/random \
-		--enable-filter-aaaa \
```

Comment:
What did that option do? Why have you removed it?

Reply:
It used to allow an option to filter out AAAA responses. It is no longer recognized by configure in this version.